packages/design-system/src/components/web-components/ds-tooltip/ds-tooltip.test.tsx

```diff
@@ -0,0 +1,152 @@
+import { act, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import './ds-tooltip';
+import './ds-tooltip-icon';
+
+jest.mock('@popperjs/core');
+
+const defaultProps = {
+  children: <ds-tooltip-icon data-testid="ds-tooltip-icon" />,
+  'class-name': 'ds-c-tooltip__trigger-icon',
+  title: 'Tooltip body content',
+  id: '1234',
+};
+
+const customTooltipText = 'Custom tooltip title text for our web component!';
+const customHeadingText = 'Custom tooltip heading text for our web component!';
+const childText = 'Bog standard child content.';
+
+const propsWithSlots = {
+  children: (
+    <>
+      <div slot="title">{customTooltipText}</div>
+      <div slot="content-heading">{customHeadingText}</div>
+      {childText}
+    </>
+  ),
+};
+
+function renderTooltip(customProps = {}) {
+  const props = { ...defaultProps, ...customProps };
+  return render(<ds-tooltip {...props} />);
+}
+
+describe('ds-tooltip', function () {
+  it('renders a tooltip', () => {
+    const { asFragment } = renderTooltip();
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('renders title and contentHeading when passed in as slots', () => {
+    renderTooltip(propsWithSlots);
+    const tooltipTrigger = screen.getByText(customTooltipText);
+    expect(tooltipTrigger).toHaveTextContent(customTooltipText);
+    expect(tooltipTrigger).toHaveTextContent(customHeadingText);
```

Comment:
Since we set both the `title` and the `contentheading` slot values, we expect both sections of text to be contained within the tooltip.